backend/handler.js

```diff
@@ -0,0 +1,32 @@
+const AWS = require("aws-sdk");
+
+module.exports.connect = async (event, context) => {
+    return {
+        statusCode: 200,
+    };
+};
+
+module.exports.disconnect = async (event, context) => {
```

Comment:
This handler is unused in your current configuration no ?

I think i'd be better to link it to the $disconnect event of the API gateway

Reply:
Yes mb, will do that